module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,43 @@
+#include "compression.hpp"
+
+#include <algorithm>
+#include <cstdint>
+
+compressedImage compressGrayscale(const Image& bitmap) {
+    std::vector<std::pair<uint8_t, uint8_t>> compressed_bitmap;
```

Comment:
You have created alias for std::vector<std::pair<uint8_t, uint8_t>>.Why not use it here?

Reply:
+1